subsys/bluetooth/host/conn.c

```diff
@@ -2122,6 +2120,43 @@ static void bt_conn_set_param_le(struct bt_conn *conn,
 	conn->le.timeout = param->timeout;
 }
 
+static bool create_param_validate(const struct bt_conn_le_create_param *param)
+{
+	/* Valid range from CONFIG_BT_CREATE_CONN_TIMEOUT Kconfig symbol */
+#if defined(CONFIG_BT_PRIVACY)
+	const s32_t timeout_max = K_SECONDS(CONFIG_BT_RPA_TIMEOUT);
+#else
+	const s32_t timeout_max = K_SECONDS(65535);
+#endif
+
+	if ((param->timeout != 0) &&
+		((param->timeout < K_SECONDS(1)) ||
+		(param->timeout > timeout_max))) {
+		return false;
+	}
```

Comment:
put a blank line after `}`

Reply:
Fixed, is space after brace a general requirement? I would have expected checkpatch to complain about it.

Reply:
It's not, rather a convention within Bluetooth code we try to keep consistent: empty line after `}`, unless the next line is also a `}` or e.g. an `#endif`.

However, there's a bigger problem with the above: you've indented the body of the branch as well as parts of the condition identically, making it hard to distinguish where the condition ends and the body starts. Split lines should be aligned so that they start right after the appropriate `(` of the preceding line (in this case the very first `(` there.